README.md

```diff
@@ -124,6 +140,30 @@ If everything is running correctly, you should get a response with information a
 }
 ```
 
+## Playground
+
+OpenFGA includes a playground use to facilitate the creation and visualisation of authorization models and tuples within your running local instance.
+
+Once OpenFGA is running, the playground can be accessed at [http://localhost:3000/playground].
+
+In the event that a port other than the default `:3000` is required, the `OPENFGA_PLAYGROUND_PORT` environment can be set when starting the instance.
+
+eg:
```

Comment:
I think this is getting confusing having so many examples of running OpenFGA. I think most people know how to set env vars, so perhaps the example can be dropped? WDYT?

Reply:
I assume nothing!

Reply:
lol. If you want to keep them can we move the `eg:` to the same line then?

> ... starting the instance. E.g.,

Reply:
Sure!